JS Heap space out of memory issue.

## Description

Build is failing actually on Master and stable. 

See this on stable https://github.com/geosolutions-it/MapStore2/actions/runs/5586073903
and this on master https://github.com/geosolutions-it/MapStore2/actions/runs/5577263753 (notice that the same PR build was successful https://github.com/geosolutions-it/MapStore2/actions/runs/5477519790 )

This can be caused by: 
- recent libraries introduced (anyway a few weeks ago, and their build has been successful for some executions)
- recent updates in nested dependencies of webpack or other NodeJS lib that cause excessive memory consumption in old versions of npm/nodeJS 

Locally using node 16.18.0 and npm 8.19.2 I have no failures. 

## How to reproduce
Build on jenkins or github actions

*Expected Result*
build successful 

*Current Result*
build cause out of memory error.

```
<s> [webpack.Progress] 92% sealing asset processing TerserPlugin
events.js:291
      throw er; // Unhandled 'error' event
      ^

Error [ERR_WORKER_OUT_OF_MEMORY]: Worker terminated due to reaching memory limit: JS heap out of memory
    at Worker.[kOnExit] (internal/worker.js:238:26)
    at Worker.<computed>.onexit (internal/worker.js:168:20)
Emitted 'error' event on Worker instance at:
    at Worker.[kOnExit] (internal/worker.js:238:12)
    at Worker.<computed>.onexit (internal/worker.js:168:20) {
  code: 'ERR_WORKER_OUT_OF_MEMORY'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mapstore2@0.6.3 fe:build: `npm run clean && mkdirp ./web/client/dist && webpack build --progress --color --config build/prod-webpack.config.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the mapstore2@0.6.3 fe:build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2023-07-18T10_12_52_271Z-debug.log
```


- [x] Not browser related

<details><summary> <b>Browser info</b> </summary>
<!-- If browser related, please compile the following table -->
<!-- If your browser is not in the list please add a new row to the table with the version -->
(use this site: <a href="https://www.whatsmybrowser.org/">https://www.whatsmybrowser.org/</a> for non expert users)

| Browser Affected | Version |
|---|---|
|Internet Explorer| |
|Edge| |
|Chrome| |
|Firefox| |
|Safari| |
</details>

## Other useful information

These failures started happening after a PR that passed the build. I think they may be caused by the old version of node/npm and recent updates to webpack nested dependencies that cause memory leaks. The past PR on master in fact passed the tests.
Locally I use node 16.18.0 and and npm 8.19.2 and it works perfectly. 

Instead here and on Jenkins we have failures, both on stable and master.

--- a/package.json
+++ b/package.json
@@ -40,6 +40,7 @@
     ]
   },
   "devDependencies": {
+    "terser": "5.18.1",
     "@babel/core": "7.8.7",
     "@babel/plugin-proposal-class-properties": "7.8.3",
     "@babel/plugin-syntax-dynamic-import": "7.8.3",
@@ -275,7 +276,7 @@
     "react-swipeable-views": "0.12.2",
     "react-textfit": "1.1.0",
     "react-widgets": "3.5.0",
-    "recharts": "0.22.4",
+    "recharts": "0.22.5",
     "recompose": "0.24.0",
     "redux": "3.6.0",
     "redux-logger": "3.0.6",

--- a/package.json
+++ b/package.json
@@ -276,7 +276,7 @@
     "react-swipeable-views": "0.12.2",
     "react-textfit": "1.1.0",
     "react-widgets": "3.5.0",
-    "recharts": "0.22.5",
+    "recharts": "0.22.4",
     "recompose": "0.24.0",
     "redux": "3.6.0",
     "redux-logger": "3.0.6",

